Neos.Media/Classes/Domain/Strategy/ThumbnailGeneratorStrategy.php

```diff
@@ -42,13 +58,21 @@ public function refresh(Thumbnail $thumbnail)
     {
         $generatorClassNames = static::getThumbnailGeneratorClassNames($this->objectManager);
         foreach ($generatorClassNames as $generator) {
-            $generator = $this->objectManager->get($generator['className']);
+            $className = $generator['className'];
+            $generator = $this->objectManager->get($className);
             if (!$generator->canRefresh($thumbnail)) {
                 continue;
             }
-            $generator->refresh($thumbnail);
-            return;
+            try {
+                $generator->refresh($thumbnail);
+                return;
+            } catch (NoThumbnailAvailableException $exception) {
+                $message = $this->throwableStorage->logThrowable($exception);
+                $this->logger->error(sprintf('%s.refresh() failed, trying next generator. %s', $className, $message), LogEnvironment::fromMethodName(__METHOD__));
```

Comment:
IMHO this should probably be debug level, but fine for now I guess?

Reply:
Isn't it a hint of an actual problem in the environment or its configuration? Then the error level would be better, so you notice that something is wrong in production.

Reply:
I guess it _can_ be something you might want to know, but on the other hand the fallback logic is implemented to allow cases were you need to pass through because of some underlaying system limitations, and in that case you will get those on a regular basis without a sensible way to get rid of them. But I guess we can adjust that after the fact.